packages/ui/src/routes/profile/+page.svelte

```diff
@@ -17,6 +17,10 @@
 	} from '$lib/services'
 	import { profile } from '$lib/stores/profile'
 
+	let y: number
+	let cls: string | undefined = undefined
+	export { cls as class }
```

Comment:
I don't think this is needed here on the `+page`. The page is displayed directly and not as a component so AFAIK there is even no way to provide some `class` to it.